src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/Util.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Net;
+using System.Security.Cryptography.X509Certificates;
+using System.ServiceModel;
+using System.ServiceModel.Web;
+using System.Text;
+using X509Certificate2 = System.Security.Cryptography.X509Certificates.X509Certificate2;
+
+
+namespace WcfService
+{
+
+    [ServiceContract]
+    public interface ICrlService
+    {
+        [OperationContract]
+        [WebGet]
+        Stream GetCrl(string serialNum);
+    }
+
+    public class CrlService : ICrlService
+    {
+        public Stream GetCrl(string serialNum)
+        {
+            // The test.crl is generated by the cert util tool and will not expire until the root cert expires.
+            // We should investigate if we can generate it in the run time. This is not a blocking issue.
+            string downloadFilePath = @"c:\\WCFTest\\test.crl";
```

Comment:
I also will open a issue to track the investigation of generate the test.crl in the runtime, thus get rid of hard coded file path.